components/mpas-ocean/src/shared/mpas_ocn_vmix.F

```diff
@@ -829,7 +829,7 @@ subroutine ocn_vel_vmix_tend_implicit_spatially_variable_mannings(meshPool, forc
            ! averaging the two bottomDepth cells.
            implicitCd = max(0.0025_RKIND, &
                         min(0.1_RKIND, &
-                          0.16_RKIND/log(250.0_RKIND*(bottomDepth(cell1)+bottomDepth(cell2)))**2 ))
+                          0.16_RKIND/(log(250.0_RKIND*(bottomDepth(cell1)+bottomDepth(cell2))))**2 ))
```

Comment:
Note, this parenthesis correction was added to this PR because `config_use_implicit_bottom_roughness` is only used in the tidal simulations used in this PR.